### PR TITLE
ascanrulesAlpha: Text4shell - Add CVE Alert Tag

### PR DESCRIPTION
**Pull request by kingthorin**
- CHANGELOG > Add change note.
- Text4ShellScanRule > Add an Alert Tag specifically for the CVE.
- Text4ShellScanRuleUniTest > Add a test to assert the expected number of alert tags.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by thc202** [comment]
There's a typo in the add-on ID (pull request and commit).

File: addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/Text4ShellScanRuleUnitTest.java [hunk 1]
```diff
@@ -113,6 +114,14 @@ void shouldStopSendingPayloadsOnNonIoException() throws Exception {
         assertThat(httpMessagesSent, hasSize(1));
     }
 
+    @Test
+    void shouldHaveExpectedNumberOfAlertRefs() {
```
Comment (thc202) [inline addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/Text4ShellScanRuleUnitTest.java:118]: `AlertTags`? (Commit/PR text would need tweaks if so.)

File: addOns/ascanrulesAlpha/CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- The Text4shell scan rules now includes an alert tag for it's CVE reference.
```
Comment (thc202) [inline addOns/ascanrulesAlpha/CHANGELOG.md:8]: `rule`

**Comment by kingthorin** [comment]
Tweaked.

**Review by thc202 — APPROVED** [review]

**Comment by ricekot** [comment]
The typo in the unit test class name in the commit message was not updated.

**Comment by kingthorin** [comment]
Got that and the double slash in the `// Given // When` ➡️  `// Given / When` comment.

**Review by ricekot — APPROVED** [review]
Thanks!

**Comment by kingthorin** [comment]
Thanks

**Pull request merged**